sklearn/multiclass.py

```diff
@@ -637,3 +649,481 @@ def predict(self, X):
         Y = np.array([_predict_binary(e, X) for e in self.estimators_]).T
         pred = euclidean_distances(Y, self.code_book_).argmin(axis=1)
         return self.classes_[pred]
+
+
+def _binomialCoeff(n, k):
```

Comment:
> > +def _binomialCoeff(n, k):

As a side remark, this naming does not abide by the coding conventions.
You might want to check the style guide:
https://www.python.org/dev/peps/pep-0008/
and make sure that your code follows it.